src/main/java/life/qbic/portal/portlet/ProjectWizardUI.java

```diff
@@ -55,7 +55,7 @@
 public class ProjectWizardUI extends QBiCPortletUI {
 
   public static boolean development = false;
-  public static boolean v3RegistrationAPI = false;
+  public static boolean v3RegistrationAPI = true;
```

Comment:
what is this?

Reply:
This setting tells the registration controller to use the openbis v3 api for sample registration. it was set to false accidentally in one of the previous PRs. Both modes work and have been tested, but the old API is very slow as it uses ETL processes.